src/lib/features/project-status/project-stale-flags-read-model/project-stale-flags-read-model.ts

```diff
@@ -7,7 +7,8 @@ export class ProjectStaleFlagsReadModel implements IProjectStaleFlagsReadModel {
     async getStaleFlagCountForProject(projectId: string): Promise<number> {
         const result = await this.db('features')
             .count()
-            .where({ project: projectId, archived: false })
+            .whereNull('archived_at')
```

Comment:
This was the actual bug. Apparently, we don't set the `archived` flag anymore, so relying on `archived = false` led to lodas of wrong results.